core/genesis/genesisBlockCreator.go

```diff
@@ -272,7 +272,7 @@ func deploySystemSmartContracts(
 		RcvAddr:   make([]byte, addrConv.AddressLen()),
 		GasPrice:  0,
 		GasLimit:  0,
-		Data:      hex.EncodeToString([]byte("deploy")) + "@" + hex.EncodeToString(factory.SystemVirtualMachine),
+		Data:      []byte(hex.EncodeToString([]byte("deploy")) + "@" + hex.EncodeToString(factory.SystemVirtualMachine)),
```

Comment:
is it ok that also "deploy" is hexEncoded? The next method "stake" is not hex.Encoded just the params

Reply:
I am not sure about this. I just cast the resulting string to a byte array. Maybe ask @sasurobert